Package.swift

```diff
@@ -1,21 +1,32 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.5
 
 import PackageDescription
 
 let package = Package(
   name: "swift-case-paths",
+  platforms: [
+    .iOS(.v13),
+    .macOS(.v10_15),
+    .tvOS(.v13),
+    .watchOS(.v6),
+  ],
   products: [
     .library(
       name: "CasePaths",
       targets: ["CasePaths"]
     )
   ],
   dependencies: [
-    .package(name: "Benchmark", url: "https://github.com/google/swift-benchmark", from: "0.1.0")
+    .package(name: "Benchmark", url: "https://github.com/google/swift-benchmark", from: "0.1.0"),
+    //.package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay.git", from: "0.5.0"),
+    .package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay.git", revision: "7bd4ee4"),
```

Comment:
will re-pin when this https://github.com/pointfreeco/xctest-dynamic-overlay/pull/42 is merged and released.